### PR TITLE
Add Navigations Keys

### PR DESCRIPTION
**Pull request by JosiahMoses**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,8 +76,75 @@
                 "command": "mind-reader.resetEditorScale",
                 "key": "shift+enter",
                 "mac": ""
-            }
-          ],
+            },
+
+			{
+				"command": "mind-reader.showAllSymboles",
```
Comment (jakergrossman) [inline package.json:82]:
```suggestion
				"command": "mind-reader.showAllSymbols",
```

File: src/commands.ts [hunk 2]
```diff
@@ -48,6 +48,67 @@ const commands: Command[] = [
     name: 'mind-reader.resetEditorScale',
     callback: resetEditorScale,
   },
+
+  //Navigation Keys......
+  {
+    name: 'mind-reader.showAllSymboles',
```
Comment (jakergrossman) [inline src/commands.ts:54]:
```suggestion
    name: 'mind-reader.showAllSymbols',
```

**Review by jakergrossman — CHANGES_REQUESTED** [review]
Looks good! Fix spelling on `showAllSymboles` and I'll approve to merge.

**Comment by JosiahMoses** [comment]
Fixed Typos in src/commands.ts and package.json

**Pull request merged**